### DISCUSSION TITLE
Where to find model when model not found (inswapper_128.onnx)

### DISCUSSION THREAD

**Discussion by ghost** [glucauze/sd-webui-faceswaplab#58] [Guide/Doc]
It looks like the devs for Roop have taken down their link for the model. Following it from the config script throws a bad gateway error.

**Comment by glucauze** [comment]
Yes you are right, i should probably disable the autodownload feature

Model seems to be available here : 
+ https://huggingface.co/deepinsight/inswapper/blob/main/inswapper_128.onnx
+ https://huggingface.co/thebiglaskowski/inswapper_128.onnx/tree/main
+ https://huggingface.co/s0md3v/inswapper_128
+ https://huggingface.co/ezioruan/inswapper_128.onnx

**Reply by ghost** [reply]
https://huggingface.co/deepinsight/inswapper/blob/main/inswapper_128.onnx

This one apparently is blocked, which is the link in the git pull. Just a heads up.

**Comment by glucauze** [comment]
I think Huggingface will gradually remove all models.  I don't want to host a model myself to respect the license. Users will have to find the model to use the extension. I'll disable the automatic download and put a warning.

I am switching to https://github.com/facefusion/facefusion-assets/releases/download/models/inswapper_128.onnx but that will be the last time i change it :)
